preshed/bloom.pxd

```diff
@@ -1,22 +1,22 @@
 from libc.stdint cimport uint64_t, uint32_t
-from cymem.cymem cimport Pool
+from libcpp.memory cimport make_unique, unique_ptr
+from libcpp.vector cimport vector
 
 ctypedef uint64_t key_t
 
-cdef struct BloomStruct:
```

Comment:
This change may look like it'd add overhead, but it's a POD in C++. Also, Cython actually translates this to a struct. So nothing changes in the compiled C++ code. However, we have to define this as a C++ class to be able to use the `new` operator (Cython doesn't understand that `new` also works with C++ structs).

Reply:
Should we add a note in the pxd file about this?

Reply:
I have changed it back to `struct`. In some iteration I replaced `new` by `make_unique`, so is works as `struct` again.